service/plugin/pom.xml

```diff
@@ -20,7 +20,6 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <snakeyaml.version>1.31</snakeyaml.version>
-        <camel.version>3.19.0</camel.version>
```

Comment:
Based on @slunelsolace comment above, would you bring back this explicit dependency for plugins?

Reply:
The plugin directory has the camel infrastructure that a plugin would use, like logging, base processors, and the ability to send data to EP via the eVMR. It's not a specific plugin implementation so I think it makes sense to keep it as is.